document-store/src/main/java/org/hypertrace/core/documentstore/model/subdoc/SubDocumentValue.java

```diff
@@ -19,6 +20,10 @@ public static SubDocumentValue of(@Nonnull final Number value) {
     return new PrimitiveSubDocumentValue(value);
   }
 
+  public static SubDocumentValue of(@Nonnull final Date date) {
```

Comment:
Any reason to expose this outside?

Reply:
To ensure that the _lastUpdateTime field is of type Date, we are exposing this method.